### PR TITLE
Fixes for Issue #4

### PR DESCRIPTION
**Pull request by PaulTFreedman**
I've attempted to implement the requested features here. Note that as part of my troubleshooting process I changed onto Bootstrap v4 - I think this was probably not necessary in the end but figured I might as well leave it on the newer version.

### CODE REVIEW

**Comment by shahzeb1** [comment]
Awesome job! 👍 

## Some feedback:
image
I'm not too familiar with Bootstrap v4 and since it seems to be in alpha, I didn't commit to using it. As the image shows, there seems to be be a breakpoint where the grid seems to break. If you can fix this grid breaking issue, then I'll go ahead and merge it!

Great job overall!

**Comment by shahzeb1** [comment]
What it currently looks like at that particular breakpoint:

image

File: mySC.js [hunk 1]
```diff
@@ -1,13 +1,22 @@
 $(document).ready(function() {
 
   // Queue up a SoundCloud song to play via the song url
-  var song = 'https://soundcloud.com/chancetherapper/acid-rain-1';
-  playSC(song, false); // false = don't autoPlay, true = autoPlay
-  getSCinfo(song);
+  var song1 = 'https://soundcloud.com/chancetherapper/acid-rain-1';
```
Comment (shahzeb1) [inline mySC.js]: Let's have a "song" array, and then loop through it.

File: mySC.js [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+  var song2 = 'https://soundcloud.com/hxns/precious';
+  var song3 = 'https://soundcloud.com/morguemami/limelight-prod-xauve';
+  var song4 = 'https://soundcloud.com/joshpanii/usedii';
+
+  playSC(song1, false); // false = don't autoPlay, true = autoPlay
+
+  getSCinfo(song1, 'thumbnail', true, true);
+  getSCinfo(song2, 'smallThumb1');
```
Comment (shahzeb1) [inline mySC.js]: Loop through these lines where the stop conditional is the number of items in our song array.

**Comment by PaulTFreedman** [comment]
@shahzeb1 I've made those changes, how does it look now? I also changed the cursor used when you hover over the extra artwork and the buttons but can easily revert if you'd rather not have that.

**Comment by shahzeb1** [comment]
Awesome! Merged.

**Pull request merged**